Clinical-Genomics/scout
# Display phenotypes

This PR adds a functionality or fixes a bug.

**How to test**:
1. Install this branch
1. Choose a variant and see that the omim phenotypes are visible

**Expected outcome**:
All OMIM phenotypes should now be visible

**Review:**
